### ISSUE TITLE
Feature request: Integrate Governance OS with Substrate API Sidecar

### ISSUE THREAD

**Issue by kenokenobingo** [NucleiStudio/governance-os#40]
Would be nice to have Governance OS integrated with [Substrate API Sidecar](https://github.com/NucleiStudio/governance-os) to be able to run an API.

**Comment by ETeissonniere** [comment]
Yes indeed, this would probably not be the Sidecar API itself as we support more than one currency in the runtime, which is not supported by sidecar yet.
